Source/Infrastructure/TextMessaging/TextMessage.cs

```diff
@@ -9,8 +9,9 @@ public class TextMessage
         public string OriginNumber { get; set; }
         public string Message { get; set; }
         public string Keyword { get; set; }
+        public string FullMessage { get; set; }
         public string ReceivedAtGatewayNumber { get; set; }
-        public double Longitude { get; set; }
-        public double Latitude { get; set; }
```

Comment:
Why are we using string representation for the lat/long values?

Reply:
We're not, I'll change it  👍